src/vs/workbench/contrib/chat/browser/chatEditing/chatEditingSession.ts

```diff
@@ -567,7 +578,41 @@ export class ChatEditingSession extends Disposable implements IChatEditingSessio
 			get result() { return responseModel.result; }
 		};
 		const entry = await this._getOrCreateModifiedFileEntry(resource, telemetryInfo);
-		entry.acceptAgentEdits(textEdits, isLastEdits);
+		entry.acceptAgentEdits(edits, isLastEdits);
+		// await this._editorService.openEditor({ resource: entry.modifiedURI, options: { inactive: true } });
+	}
+
+	private async _acceptNotebookEdits(resource: URI, edits: ICellEditOperation[], isLastEdits: boolean, responseModel: IChatResponseModel): Promise<void> {
+		// if (this._filesToSkipCreating.has(resource)) {
```

Comment:
This is a TODO item, not done in this PR, as we're only focused on adding the API layer changes.